### DISCUSSION TITLE
High overlap between CD8 CD68 CD3

### DISCUSSION THREAD

**Discussion by Katitamorais** [IBEXImagingCommunity/ibex_imaging_knowledge_base#240] [Q&A]
Hello!

We are running a cycle combining CD8 CD68 CD3. We got good fluorescence signal and specificity using FFPE. However, when we used fixed frozen tissue (1:4 Cytofix/Cytoperm Fixed Frozen), there is a high overlap between CD8 CD68 CD3.

![CD3 CD8 CD68 IBEX FFPE fixed frozen](https://github.com/user-attachments/assets/818eacd4-849b-41ea-9547-b608fb3adaf1)

CD3 - clone EPR4426 - validated for IBEX FPPE -https://www.abcam.com/en-us/products/primary-antibodies/alexa-fluor-555-cd3d-antibody-ep4426-ab208514
CD68 - clone KP1 - validated for IBEX FFPE and Cytofix/Cytoperm - https://www.capricobio.com/product/cd68-kp1-ifluor-594/
CD8 - clone C8/144B - validated for FFPE - https://www.biolegend.com/de-de/products/alexa-fluor-647-anti-human-cd8a-antibody-14127

Interestingly, when CD8 and CD4 were evaluated together in fixed frozen tissue, we could see specificity.

![CD8 CD4 IBEX colon fixed frozen](https://github.com/user-attachments/assets/7ed1657d-1951-4f2e-a072-f95a6b79d7dc)

Now we will run a test simultaneously evaluating 4 tissues types and both FFPE and fixed frozen tissue.

Have you already seen anything similar?

Thanks.
Kátia.

**Comment by radtkea** [comment]
Hi Katia,

Thank you so much for asking your question in the open! Apologies for my delay in answering. You can always @radtkea to get me directly.

Thank you for providing the images and clones you are using.

My answer: you need to use different clones/antibodies for the fixed frozen tissues. There are a few clones (CD68 clone KP1) that work in both fixed frozen and FFPE tissues. You can search for appropriate clones in the reagent resources table by selecting "1:4 Cytofix/Cytoperm Fixed Frozen" in the tissue preservation method tab. You rightly noted this in your answer above!

For CD4, I recommend the RPA-T4 clone. It's available in many different conjugates from different vendors. [AF647](https://ibeximagingcommunity.github.io/ibex_imaging_knowledge_base/supporting_material/CD4_AF647/0000-0003-4379-8967.html) is a great choice. 

For CD3, I recommend the UCHT1 clone. It's also available in many different conjugates from different vendors. [iF594](https://ibeximagingcommunity.github.io/ibex_imaging_knowledge_base/supporting_material/CD3_iF594/0000-0003-4379-8967.html) was used throughout our studies.

For CD8, I recommend the SK1 clone. I used it mostly in [AF488](https://ibeximagingcommunity.github.io/ibex_imaging_knowledge_base/supporting_material/CD8_AF488/0000-0003-4379-8967.html) and used a 1:25-1:40 dilution.

I have not used the CD3 and CD8 clones you specified for fixed frozen tissue. As to why they might work together "separately" but not together is spectral overlap: the CD68 iF594 might be spilling into the AF555 and AF647 channels. This is the one clone (KP1) validated for both tissue preservation methods. If you titrate it, then you might be able to see the CD3 and CD8 better.  

I would avoid PE in the colon. It appears to bind non-specifically in gut tissue.

Best of luck! Please reach out if you need anything else.
Andrea

**Comment by Katitamorais** [comment]
Hi @radtkea,

I just want to confirm if I understood correctly: we may not always be able to use the same antibody panel for FFPE and Fixed Frozen Tissue, meaning that in some cases, specific clones might be required depending on the tissue preservation method.

Regarding PE-conjugated antibodies, is there any publication or reference you would recommend for consultation? We have obtained good results using CD4-PE (clone RPA-T4) using Fixed Frozen Tissue, with strong fluorescence signals and high specificity (no co-expression with CD8). For FFPE, we got good results using the clones EPR6855 and SK3.

Additionally, we performed simultaneous tests on different tissue types. In FFPE (Tonsil and Colon Cancer) tissues and Fixed Frozen Ovary and Cervical Cancer samples, we observed some CD8+CD68+ cells, with better results than the previous images posted here. However, in Fixed Frozen Colon Cancer tissues, the frequency of these double-positive cells seems to be higher than in the other tissue types.

<img width="1993" alt="image" src="https://github.com/user-attachments/assets/b9ae188a-61ee-4042-bd13-5bc3c0162a95" />


We are using the LSM 710 confocal microscope (Zeiss), and we have carefully verified that CD68 iF594 is not spilling into the AF555 or AF647 channels.

As soon as we gather more data, we will share the findings here.

Thank you for your support!

Best regards,
Katia.

**Reply by radtkea** [reply]
Hi Katia,

Great to hear from you! Thank you for sharing your results with us! Your figures are coming together nicely!

**Clones in FFPE versus Fixed Frozen:** 
Yes, you got it! In the lab I had distinct antibody inventories: mouse fixed frozen, human fixed frozen, and human FFPE. Secondary antibodies could be used for all sample preparations. This year @zivy and I would like to report high usage clones by tissue preservation method and animal. Until then, happy to help you build out your antibody panels depending on your markers of interest.

**CD4 clones**
Yes, this [CD4 clone](https://www.abcam.com/en-us/products/primary-antibodies/cd4-antibody-epr6855-ab133616?srsltid=AfmBOooqaZ_cVLt8atKeFu_yWQYd-dw351Uo8-YmkRXh0J4uYy00znIc) from Abcam is great in FFPE tissues. I have never used clone SK3 before. Glad it works in FFPE tissues. I believe it's a popular flow cytometry clone. 

**PE reference**
Here's a [paper ](https://onlinelibrary.wiley.com/doi/full/10.1002/cyto.a.22276#:~:text=For%20example%2C%20IgM%20antibody%20has,vivo%20using%20germ%2Dfree%20mice.) describing non-specific binding of PE-conjugated antibodies to plasma cells. 
My terrific post-bac, Evelyn, went down a rabbit hole regarding the structural similarities between bilirubin and PE. 

**Time to discuss data**
Happy to find a time to review your data with you. Just reach out via email: andrea.radtke "@" leica-microsystems.com or via my gmail, which you have. 

**Spectral overlap**
Glad you don't have spectral overlap. Ah yes, the 710! The 710 was the first microscope I ever used as a PhD student at Hopkins and also in the Germain lab to do intravital microscopy at the NIH. Fond memories.

Best wishes,
Andrea
